test/e2e/app-dir/use-cache-hanging-inputs/app/uncached-promise-nested/page.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import { setTimeout } from 'timers/promises'
 
 async function getUncachedData() {
-  await setTimeout(0)
```

Comment:
The refactoring uncovered an issue which is a combination of tracking chunk loads with the cache signal now (via #74152), and how our hanging input signal heuristic is designed. With Turbopack, this example is not detected as a hanging promise right now, because there are pending chunk loads tracked by the cache signal that we await before aborting the input serialization. We might want to revisit this in a follow-up, not sure how though. /cc @lubieowoce